### PR TITLE
Skip all grib tests for Python 3.

### PR DESCRIPTION
**Pull request by pp-mo**
**this is now a placeholder: DO NOT MERGE**
see : https://github.com/SciTools/iris/pull/1833#issuecomment-157074499

~~This excludes the grib tests from the standard test run in Python 3.
This is needed because we decided not to support Grib under Python 3 for the present,
because of outstanding problems (see https://github.com/SciTools/iris/pull/1806).~~

~~TO TEST:
running ```python3 setup.py test``` will generate various grib-related errors, which this removes.
Unfortunately, all tests won't run successfully under Python 3, until various other outstanding problems are resolved (see list in #1782)~~

### CODE REVIEW

**Comment by QuLogic** [comment]
Looks good to me.

File: lib/iris/tests/__init__.py [hunk 1]
```diff
@@ -810,8 +810,14 @@ class MyPlotTests(test.GraphicsTest):
     return skip(fn)
 
 
-skip_grib = unittest.skipIf(not GRIB_AVAILABLE, 'Test(s) require "gribapi", '
-                                                'which is not available.')
+if six.PY3:
+    skip_grib = unittest.skipIf(
+        True,
+        'Test(s) use "grib", which is not currently supported under Python 3.')
```
Comment (rhattersley) [inline lib/iris/tests/__init__.py]: Why `grib` instead of `gribapi`?

**Comment by pp-mo** [comment]
> @rhattersley  Why grib instead of gribapi?

Good point --  I think I wasn't quite clear in my mind **what** it is that we have problems with in Python 3.
Fixed.

**Comment by pp-mo** [comment]
> what it is that we have problems with in Python 3.

Following offline discussion with @marqh ..
**It now seems we don't need this after all.**

Rather than disable Iris grib usage in Python 3 (which only awaits a Python-3 functional gribapi), we have now *instead* removed the Python3 gribapi build from the scitools channel.
 * [conda-recipes-scitools #118](https://github.com/SciTools/conda-recipes-scitools/pull/118)
 * [specific commit](https://github.com/pelson/conda-recipes-scitools/commit/61673d154106cff176f9af83e4f613e1f078c14f)

So there is then no need to treat this as a special case :  When installed under Python3 you have no gribapi, so it automatically skips the tests.

**Comment by pp-mo** [comment]
>  it automatically skips the tests.

... except it isn't working, for me.
I think an iris.proxy solution may be getting in the way.
So something may still be needed here.

**Comment by QuLogic** [comment]
Are you sure it isn't working? I just ran all tests without `gribapi` installed on Python 3.4 without any failures, except the one example that requires `gribapi` and isn't automatically skipped.

**Comment by pp-mo** [comment]
> Are you sure it isn't working?

I think the  problem may be local to my installation:
my conda is refusing to "forget" the gribapi for Python3 that it used to have

I expect you'll get #1842 "Enable Python 3" functioning any time soon.
If so, that renders this obsolete + I'll close it.

**Comment by pp-mo** [comment]
Dead, not needed now that #1842 has gone in.

**Pull request closed without merging**